crates/sparrow-session/src/session.rs

```diff
@@ -503,6 +544,49 @@ impl Session {
             schema,
         ))
     }
+
+    pub fn plan(
```

Comment:
Hm, the verb `plan` sounds like it could be in the execution path, rather than an option to get the string representation of the plan. Maybe would prefer more explicit `get_string_for_plan`, or something like that, but :shrug:

Reply:
Actually, the best option might be `explain`:

https://duckdb.org/docs/guides/meta/explain.html

https://www.geeksforgeeks.org/explain-in-sql/

Different SQLs use different ways to "explain" the query, but that seems like what this is. Basically, you can ask us to "explain" what we'll do to compute something in various ways.

Reply:
Ah, nice. Yeah, `explain` then

Reply:
Changed to `explain`.